src/Atc.DotNet/DotnetHelper.cs

```diff
@@ -29,7 +29,7 @@ public static DirectoryInfo GetDotnetDirectory()
                 return directory;
             }
 
-            // The environment variable 'DOTNET_ROOT' specifies the location of the .NET runtimes,
+            // The environment variable 'DOTNET_ROOT' specifies the location of the .NET runtime,
```

Comment:
"runtimes" is how Microsoft refers to it in https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables. Probably due to the folder containing all installed versions of dotnet